src/thor/triplegbuilder.cc

```diff
@@ -534,9 +534,7 @@ void SetTripEdgeRoadClass(TripLeg_Edge* trip_edge,
  * @param  trip_node          Trip node to add the edge information to.
  * @param  graphtile          Graph tile for accessing data.
  * @param  second_of_week     The time, from the beginning of the week in seconds at which
- *                               the path entered this edge
- * @param  length_percentage  Scale for the edge length for the partial distance
- *                               at begin and end edges
```

Comment:
this was unfortunate because we add the edge before we add the shape. since we delay knowing how much to trim until we look at the shape it made sense to just remove setting the edge length from here and do it afterwards